docs/design/selector-generation.md

```diff
@@ -110,18 +110,42 @@ No changes
 ## Kubectl
 
 No required changes.
-Suggest moving SELECTOR to wide output of `kubectl get jobs` since users don't write the selector.
+Suggest moving SELECTOR to wide output of `kubectl get jobs` since users do not write the selector.
 
 ## Docs
 
 Remove examples that use selector and remove labels from pod templates.
 Recommend `kubectl get jobs -l job-name=name` as the way to find pods of a job.
 
-# Cross Version Compat
+# Conversion
 
-`v1beta1` will not have a `job.spec.manualSelector` and will not provide a default selector.
+The following applies to Job, as well as to other types that adopt this pattern.
+
+Type `extensions/v1beta1` will not have a `job.spec.manualSelector`.  Both the internal type and the `batch/v1` type will have
```

Comment:
I can name it "autoSelector", so that it as the opposite meaning, and its absence has the existing v1beta1 behavior. Also, no annotation needed.  Sound good?